modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ObjectDefinitionExportImportTest.java

```diff
@@ -58,8 +58,13 @@ public void testExportImportObjectDefinition() throws Exception {
 			objectDefinition.getExternalReferenceCode(), "AccountEntry");
 
 		testExportImport(
-			"test-object-definition.json", "test-object-definition.json", null,
-			"TestObjectDefinition");
+			"test-object-definition-1.json", "test-object-definition-1.json",
+			null, "TestObjectDefinition1");
+
+		testExportImport(
+			"test-object-definition-2.json",
+			"test-object-definition-2-expected.json", null,
```

Comment:
Can you change the file name to `test-object-definition-2.system-en-labels.json`